src/ppx/grfx/vk/vk_render_pass.cpp

```diff
@@ -26,6 +26,7 @@ namespace vk {
 
 Result RenderPass::CreateRenderPass(const grfx::internal::RenderPassCreateInfo* pCreateInfo)
 {
+    bool hasMultiView   = ToApi(GetDevice())->HasMultiView() ? true : false;
```

Comment:
`HasMultiView()` already returns a boolean, right? Is there a reason to add the ternary operator?